boards/arm/efr32mg_sltb004a/board.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2018 Christian Taedcke, Diego Sueiro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef _ASMLANGUAGE
+
+#include <soc.h>
+
+#endif /* !_ASMLANGUAGE */
+
+#ifdef __cplusplus
+}
+#endif
+
+/* Environment Sensors Enable Pin */
+#define ENV_SENSE_ENABLE_NAME	CONFIG_GPIO_GECKO_PORTF_NAME
```

Comment:
where is this used?

Reply:
This one I used for testing an i2c sensor in the board to validate the i2c driver.
I'll remove it.

Reply:
Done